webpack.config.js

```diff
@@ -25,67 +24,119 @@ export default {
     hot: true
   },
   plugins: [
-    new webpack.DefinePlugin(GLOBALS),
+    new webpack.DefinePlugin(GLOBALS), //define variables to use on development
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NoEmitOnErrorsPlugin(),
     new webpack.LoaderOptionsPlugin({
       debug: true
     })
   ],
   module: {
-    rules: [
-      {
-        test: /\.js$/,
-        include: path.join(__dirname, 'app'),
-        use: [{
-          loader: 'babel-loader'
-        }]
-      },
-      {
-        test: /(\.css)$/,
-        use: [{
-          loader: 'style-loader'
-        },
-        {
-          loader: 'css-loader'
-        }]
-      },
-      {
-        test: /\.eot(\?v=\d+\.\d+\.\d+)?$/,
-        use: [{
-          loader: 'file-loader'
-        }]
-      },
-      {
-        test: /\.(woff|woff2)$/,
-        use: [{
-          loader: 'url-loader',
-          options: {
-            prefix: 'font',
-            limit: 5000
-          }
-        }]
-      },
-      {
-        test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/,
-        use: [{
-          loader: 'url-loader',
-          options: {
-            limit: 10000,
-            mimetype: 'application/octet-stream'
+    rules: [{
+      test: /\.js$/,
+      include: path.join(__dirname, 'app'),
+      use: [{
+        loader: 'babel-loader'
+      }]
+    }, {
+      enforce: 'pre',
+      test: /\.scss/,
+      loader: 'import-glob-loader'
+    }, {
+      test: /(\.css)$/,
+      use: [{
+        loader: 'style-loader'
+      }, {
+        loader: 'css-loader'
+      }]
+    }, {
+      test: /(\.scss)$/,
+      use: [{
+        loader: 'style-loader'
+      }, {
+        loader: 'css-loader'
+      }, {
+        loader: 'postcss-loader',
+        options: {
+          plugins: function () {
+            return [
+              require('autoprefixer')
+            ];
           }
-        }]
-      },
-      {
-        test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
-        use: [{
-          loader: 'url-loader',
+        }
+      }, {
+        loader: 'sass-loader'
+      }]
+    }, {
+      test: /\.eot(\?v=\d+\.\d+\.\d+)?$/,
+      use: [{
+        loader: 'file-loader'
+      }]
+    }, {
+      test: /\.font\.(js|json)$/,
+      use: [{
+        loader: 'style-loader'
+      }, {
+        loader: 'css-loader'
+      }, {
+        loader: 'webfonts-loader'
```

Comment:
Asking out of ignorance: is `css-loader` required if we use `webfonts-loader`?

Reply:
Idem above 👍